core/deployment/src/main/java/io/quarkus/deployment/steps/NativeImageFeatureStep.java

```diff
@@ -53,7 +53,7 @@ void addExportsToNativeImage(BuildProducer<JPMSExportBuildItem> features) {
         features.produce(new JPMSExportBuildItem("org.graalvm.sdk", "org.graalvm.nativeimage.impl", null,
                 GraalVM.Version.VERSION_23_1_0));
         features.produce(new JPMSExportBuildItem("org.graalvm.nativeimage", "org.graalvm.nativeimage.impl",
-                GraalVM.Version.VERSION_23_0_0));
+                GraalVM.Version.VERSION_23_1_0));
```

Comment:
I'm surprised it's the only one that needs changing? I would have expected all the current entries to be adjusted?

Reply:
This is the only place the constructor setting the `exportSince` is used. Thankfully we don't have to export many things :)